ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/default_interface_mtu_platform.yml

```diff
@@ -9,3 +9,6 @@ default_interface_mtu: 8765 # This will not be used since platform settings take
 platform_settings:
   - platforms: [ default ]
     default_interface_mtu: 1234
+    structured_config:
```

Comment:
Add a comment above stating this is the test for `structured_config`. It may be better to create another host (though it feels like a waste of a host here). Maybe we could do something like having a host called `platform` and use a folder directory with one file called `default_interface_mtu.yml` and one called `structured_config.yml` so that it is clear in the file name what we are checking.

@carlbuchmann  / @ClausHolbechArista - thoughts?

Reply:
Just create a new host like `platform_settings` so we can add other tests there.